src/api/Feed.js

```diff
@@ -184,14 +185,73 @@ export const getParsedFileActivitiesResponse = (response?: { entries: FileActivi
                     return { ...source[FILE_ACTIVITY_TYPE_APP_ACTIVITY] };
                 }
 
+                case FILE_ACTIVITY_TYPE_VERSION: {
+                    const versionsItem = { ...source[FILE_ACTIVITY_TYPE_VERSION] };
+
+                    versionsItem.type = 'file_version';
```

Comment:
you can move this to inside the declaration of the object. unless it's cleaner to keep it outside so it remains consistent with the other cases

Reply:
I will probably keep it consistent